client/src/utils/validator.ts

```diff
@@ -0,0 +1,37 @@
+import { ERRORS } from '../constants/errors';
+import { VALIDATORS_STRINGS } from '../constants/validatorStrings';
+import { VALIDATOR } from '../constants/validator';
+
+interface validatorsArray {
+  valName: string;
+  param: string;
+}
+const createInputValidators = (name: string) => {
+  const validatorsNew: validatorsArray[] = [];
+  VALIDATORS_STRINGS[name].map((validator: string) => {
+    const separator: string | undefined = '(';
+    const valName: string = validator.split(separator)[0];
+    const regExp = /[()]/;
+    const param: string = validator.split(regExp)[1];
+    validatorsNew.push({
+      valName,
+      param
+    });
+    return null;
+  });
+  return validatorsNew;
```

Comment:
`map` method is used to create new array with the same length, based on array is called with. in your case this method is used incorrectly: you should either return new item (you're returning null) or use `forEach` method instead of `map`, because `forEach` doesn't require `return` statement. i'd recommend to use `map`, but use it in correct way